packages/customer-account-ui-extensions-react/src/hooks/tests/api.test.tsx

```diff
@@ -0,0 +1,29 @@
+import type {
+  StandardApi,
+  FullPageApi,
+} from '@shopify/customer-account-ui-extensions';
+import {useApi} from '../api';
+
+import {mount} from './mount';
+
+describe('useApi', () => {
+  it('returns api', async () => {
+    const extensionApi = {} as StandardApi & FullPageApi;
```

Comment:
@emileber @robin-drexler could you look at this test one more time? I noticed that there is a type issue here and copy&pasting from checkout wasn't enough. The extension points are typed differently in checkout (they are all generics) and they are using some recursive `DeepPartial` typing mechanism to make it work. We don't :)

Reply:
Looks fine! You possibly provide more fields in the mock API to make sure we match against the same object, and not some random empty object.

Reply:
Ah since I'm casting it anyways, I don't have to care that all the properties are mandatory right? I could just provide a few since I'm overriding it with the cast.

Reply:
Yes exactly